src/swagger.json

```diff
@@ -132,41 +147,6 @@
         }
       }
     }
-  },
-  "components": {
-    "schemas": {
-      "User": {
-        "type": "object",
-        "properties": {
-          "name": {
-            "type": "string",
-            "description": "The name of the user."
-          },
-          "email": {
-            "type": "string",
-            "description": "The email address of the user."
-          },
-          "phone": {
-            "type": "string",
-            "description": "The phone number of the user."
-          },
-          "address": {
-            "type": "string",
-            "description": "The address of the user."
-          },
-          "password": {
-            "type": "string",
-            "description": "The password of the user."
-          }
-        },
-        "required": [
-          "name",
-          "email",
-          "phone",
-          "address",
-          "password"
-        ]
-      }
-    }
   }
 }
```

Comment:
You have deleted alot

Reply:
he was trying to show sample and we decided as team to remove it everything is fine